From 77b7a28fbfadac38fae14a5f1fec97ade2fc8a85 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Wed, 31 Oct 2018 18:11:33 +0800
Subject: [PATCH] phy: add PCIE_EP and PCIE_RC modes

Innosilicon combophy for PCIe still need different
configuration between EP and RC mode.

Change-Id: Ie1f14e63785f44d84a2b3a154990c6a54eb1156e
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 include/linux/phy/phy.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/phy/phy.h b/include/linux/phy/phy.h
index a3965c3cc653..43b2da6f4be2 100644
--- a/include/linux/phy/phy.h
+++ b/include/linux/phy/phy.h
@@ -24,6 +24,8 @@ struct phy;
 
 enum phy_mode {
 	PHY_MODE_INVALID,
+	PHY_MODE_PCIE_EP,
+	PHY_MODE_PCIE_RC,
 	PHY_MODE_USB_HOST,
 	PHY_MODE_USB_DEVICE,
 	PHY_MODE_USB_OTG,
-- 
2.35.3

